setup.py

```diff
@@ -19,15 +19,13 @@
     packages=['itk'],
     package_dir={'itk': 'itk'},
     download_url=r'https://github.com/KitwareMedical/ITKMorphologicalContourInterpolation',
-    description=r'Interslice interpolations of label images based on their morphology .',
-    long_description='ITK is an open-source, cross-platform library that '
-                     'provides developers with an extensive suite of software '
-                     'tools for image analysis. This package implements morphological contour interpolation. '
-                     'For more information, see  '
+    description=r'Image morphological contour interpolation.',
+    long_description='itk-morphologicalcontourinterpolation provides classes '
+                     'to perform image morphological contour interpolation.\n'
+                     'Please refer to:\n'
                      'Zukić D., Vicory J., McCormick M., Wisse L., Gerig G., Yushkevich P., Aylward S. '
```

Comment:
I prefer Zukić Dž. as abbreviation, due to it being considered a [single letter](https://en.wikipedia.org/wiki/D%C5%BE) in my native alphabet.

Reply:
Done in a12554a.